test/fixtures/default/arista4-32a

```diff
@@ -0,0 +1,20 @@
+---
+arguments_string: arista4-32a
+environment_variables: {}
+expected_hash:
+  HOSTS:
+    arista4-32-1:
+      pe_dir: 
+      pe_ver: 
+      pe_upgrade_dir: 
+      pe_upgrade_ver: 
+      hypervisor: vmpooler
+      platform: eos-4-i386
+      template: arista-4-i386
+      roles:
+      - agent
+  CONFIG:
+    nfs_server: none
+    consoleport: 443
+    pooling_api: http://vmpooler.delivery.puppetlabs.net/
+expected_exception: 
```

Comment:
should all these fixtures be checked in?  are these not what are being generated by the code above?